include/odp/autoheader_external.h.in

```diff
@@ -11,4 +11,7 @@
 /* Define cache line size */
```

Comment:
In 39321eb3d647a857ed0e245f14e4171e430e3ebf message, there could be a whitespace before `odp_packet_free_sp()`.

Reply:
Fixed in V4.